README.md

```diff
@@ -1,20 +1,43 @@
 # \*IPs - Star Improvement Proposals
 *by Zachary Belford, Shane Jonas, Donald McIntyre, Yaz Khoury, Anthony Lusardi, Tomaz Kariz*
 
-Star Improvement Proposals are an open standard for contributors to propose changes, additions, or deprecations of core protocol specifications.
+Star Improvement Proposals is an extension of the BIP/EIP/ECIP process, with some changes to allow for decentralization of the process. Its fundamental change involves determinism of proposal numbers and content-based versioning. 
 
 The Star is meant to be replaced with the prefix/accronym of the users choice, though the prefix MUST be the same for all participants of the particular protocol specification.
 
 For example, All forks of the Star Improvement Proposal relating to the bitcoin network MUST be called `BIP`. The differentiator being the location/domain/github organization to which the \*IP belongs.
 
-Star Improvement Proposals are versioned by the content of their name. That is, in the case of making a proposal, the `version` identifier of the proposal MUST be derived by the last 8 characters of the SHA-3-512 of the proposal name. This `version` naming scheme is deterministic in that groups or individuals may choose to include another's proposal without any numbering conflicts.
+# Changes to BIP/EIP/ECIP
+## Version
+Star Improvement Proposals are versioned by the content of their name. First we create a hash of the preamble, then we use a substring of it to create the version hash.
 
-Once you have your \*ip repo up, people submit proposals to it via pull requests, adding a file named {networkName}IP-{proposalName}-{version}. Once the proposal is implemented by your team / group, you may signal your readyness by merging the pull request.
+### Preable hash
+SHA-3-512 of the proposal [preamble](https://github.com/ethereumclassic/ECIPs/blob/master/ECIP-1.sample.md#what-is-an-ecip).
```

Comment:
It's not clear to me what a "preamble" is. Does it include the title and the metadata?

From the example link:

Is it this?
```
### What is an ECIP?

    ECIP: 1
    Title: ECIP Purpose and Guidelines
    Status: Draft
    Type: Meta
    Author: Your Name <your.email@domain.com>
    Created: 2017-11-30
```

Is it just this?
```
    ECIP: 1
    Title: ECIP Purpose and Guidelines
    Status: Draft
    Type: Meta
    Author: Your Name <your.email@domain.com>
    Created: 2017-11-30
```

Reply:
Have just found https://github.com/BelfordZ/starIPs/issues/9, which talks about this. Moving questions there.

Reply:
Hi @whilei the idea of the startIP system is to create a deterministic numbering so that all ECUPs can be decentralized in different repos, but with a common standard.

These are examples of centralized preambles in BIP, EIP, ECIP: 

BIP:
```
 BIP: <BIP number, or "?" before being assigned>
* Layer: <Consensus (soft fork) | Consensus (hard fork) | Peer Services | API/RPC | Applications>
  Title: <BIP title; maximum 44 characters>
  Author: <list of authors' real names and email addrs>
* Discussions-To: <email address>
* Comments-Summary: <summary tone>
  Comments-URI: <links to wiki page for comments>
  Status: <Draft | Active | Proposed | Deferred | Rejected |
           Withdrawn | Final | Replaced | Obsolete>
  Type: <Standards Track | Informational | Process>
  Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
  License: <abbreviation for approved license(s)>
* License-Code: <abbreviation for code under different approved license(s)>
* Post-History: <dates of postings to bitcoin mailing list, or link to thread in mailing list archive>
* Requires: <BIP number(s)>
* Replaces: <BIP number>
* Superseded-By: <BIP number>
```

EIP:
```
eip: (this is determined by the EIP editor)
title:
author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s). Details are below.>
* discussions-to: <a url pointing to the official discussion thread>
status: <Draft | Last Call | Accepted | Final | Active | Deferred | Rejected | Superseded>
* review-period-end:
type: <Standards Track (Core, Networking, Interface, ERC) | Informational | Meta>
* category: <Core | Networking | Interface | ERC>
created:
* updated:
* requires: <EIP number(s)>
* replaces: <EIP number(s)>
* superseded-by: <EIP number(s)>
* resolution: <a url pointing to the resolution of this EIP>
```

ECIP:
```
 ECIP: <ECIP number>
  Title: <ECIP title>
  Author: <list of authors' real names and optionally, email address>
* Discussions-To: <email address>
  Status: <Draft | Active | Accepted | Deferred | Rejected |
           Withdrawn | Final | Superseded>
  Type: <Standards Track | Informational  Meta>
  Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
* Replaces: <ECIP number>
* Superseded-By: <ECIP number>
* Resolution: <url>
```

Reply:
Resolving this conversation as it is discussed here: https://github.com/BelfordZ/starIPs/issues/9